bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/api-bridge-type.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="freeboxos"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="api">
+		<label>Freebox OS Api</label>
+		<description>Bridge between hosts and the API rest service</description>
+
+		<representation-property>hostAddress</representation-property>
```

Comment:
Normally, you should refer a configuration parameter here. I don't find the parameter "hostAddress".

Reply:
Well spotted, this configuration item name changed.